src/pages/case-study/doctorlink-setting-up-a-design-system.js

```diff
@@ -0,0 +1,578 @@
+import React, { Fragment } from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+// import { Padding } from 'styled-components-spacing'
+import ReactMarkdown from 'react-markdown'
+// import generate from 'shortid'
+
+import { Grid, Row, Col } from '../../components/grid'
+import { SectionTitle, BodyPrimary } from '../../components/Typography'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import Layout from '../../components/layout'
+// import { makeText } from '../../utils/makeText'
+import Head from '../../components/Common/Head'
+// import Image from '../../components/Common/Image'
+import VideoSection from '../../components/Common/VideoSection'
+// import SubtitleWithBody from '../../components/Common/SubtitleWithBody'
+// import Statement from '../../components/Common/Statement'
+import BlueBackground from '../../components/Common/BlueBackground'
+import GreyBackground from '../../components/Common/GreyBackground'
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+
+// const PaddingBottomGrid = styled(Grid)`
+//   padding-bottom: ${({ theme }) => theme.space[7]};
+// `
+
+const BlockRow = styled(Row)`
```

Comment:
I understand what you're trying to do with this BlockRow component but ultimately it's going to cause you more  issues in the long run.

Please make specific rows for each section, it'll be a lot of components I know but due to the nature of the page, the design is not going to change a huge amount. It's not adding complexity to the page, it's just adding single layer wrappers, of course you end up with more components but it's a lot safer. It's better practise in this situation to create specific components here instead of trying to create a one size fits all solution. 

Due to having this BlockRow component wrapping all of the blocks, one change here has an effect on 32 sections.

Reply:
Cool. Will do then 👍